action.yml

```diff
@@ -328,6 +332,23 @@ runs:
                   '\n[Full pipeline log](${{ steps.artifacts_url.outputs.url }}/${{ steps.sched_test.outputs.req_id }}/pipeline.log).'
           })
 
+    - name: Create github summary
+      if: ${{ inputs.create_github_summary == 'true' }}
+      shell: bash
+      run: |
+        if [ -z "${{ steps.final_state.outputs.INFRA_STATE }}" ]; then
+          infra_state=OK
+        else
+          infra_state=Failed
+        fi
+        if [ -z "$GITHUB_SUMMARY" ]; then
+          echo "| Compose | Arch | Infrastructure State | Test result | Link to logs |" > "$GITHUB_STEP_SUMMARY"
```

Comment:
`Compose` is what? in our case it is `RHEL7 - 12`?

Reply:
Compose is only a compose. In the example you've mentioned it would be RHEL7.

Reply:
And is TFaGA specific only to containers? I think, that also other code, non related to containers can be tested with TFaGA.

Reply:
Like my comment above, what about if the $GITHUB_SUMMARY is not empty? User can add there what ever he / she wants and then the header is not entered.

Reply:
Yes, you are right. This needs to be reworked.

Reply:
Resolved in [eca59db](https://github.com/sclorg/testing-farm-as-github-action/pull/74/commits/eca59dbf243e3a6443a694a129312db971b024d5).